src/coord/Cargo.toml

```diff
@@ -22,8 +22,11 @@ log = "0.4"
 ore = { path = "../ore" }
 pgrepr = { path = "../pgrepr" }
 repr = { path = "../repr" }
-rusqlite = { version = "0.20", features = ["bundled"] }
 rdkafka = { version = "0.23.1", features = ["cmake-build"] }
+rusoto_core = "0.42.0"
+rusoto_credential = "0.42.0"
+rusoto_kinesis = "0.42.0"
```

Comment:
Can you upgrade all of these to 0.43.0-beta.1? Normally using the stable version would absolutely be the right call, but 0.42.0 is still on the Tokio 0.1 ecosystem, and it brings in a metric ton of dependencies. Given that we're still beta ourselves, In this case I think it makes sense to tolerate some instability in exchange for a smaller binary and faster compiles.

Reply:
Yes! Thanks for pointing that out